src/cpu/aarch64/jit_brgemm_conv.cpp

```diff
@@ -634,7 +634,7 @@ status_t brgemm_convolution_fwd_t<isa, use_inversion>::add_po_kernel(
     bcfg->alpha = !is_init && IMPLICATION(jcp.with_sum, jcp.use_buffer);
     bcfg->beta = is_init ? 0 : 1;
     CHECK(safe_ptr_assign(kernels_po_[ker_idx],
-            new jit_brgemm_kernel_post_ops(jcp, *bcfg, *_pd->attr())));
+            new jit_brgemm_kernel_post_ops<isa>(jcp, *bcfg, *_pd->attr())));
```

Comment:
I very highly encourage not to step on template path and pass length as argument to the function that crates a kernel or to a proxy that will dispatch templated kernel.

Reply:
Thank you for the feedback. Currently, we have implemented this using the template path. We will look into updating it to pass the length as an argument in future revisions.